pinot-core/src/main/java/com/linkedin/pinot/core/operator/docidsets/SizeBasedDocIdSet.java

```diff
@@ -39,12 +38,10 @@ public int getMaxDocId() {
 
   @Override
   public void setStartDocId(int startDocId) {
-    throw new UnsupportedOperationException();
```

Comment:
why did you remove it, its ok for docIdSets to be immutable, i.e the min and max doc id cannot be changed later.

Reply:
They should not be changed, but calling it should not throw exception. That's why I leave it blank (no-op)